styleguide/derek/pattern-components/lp-callout/_lp-callout.scss

```diff
@@ -70,3 +70,78 @@
     padding: 15px 25px 35px;
   }
 }
+
+.lp-legal-header {
+  display: block;
+  font-size: 16px;
+  font-weight: 600;
+  margin-bottom: 5px;
+}
+
+.lp-legal-p {
+  font-size: 14px;
+  font-style: italic;
+}
+
+.lp-relatedResource-card-3 {
+  @include make-md-column(4);
+
+  .lp-relatedResource-img {
```

Comment:
Can we a take more of a MaintainableCSS approach to this?  

`.lp-relatedResource-card3Img`

This seems like nesting we can avoid.